pkg/reactable/reactable.go

```diff
@@ -10,8 +11,10 @@ import (
 	"github.com/genshinsim/gcsim/pkg/core/info"
 )
 
+const maxChars = 4
+
 type Reactable struct {
-	Durability [info.ReactionModKeyEnd]info.Durability
+	Durability [info.ReactionModKeyEnd][maxChars]info.Durability
```

Comment:
move `maxChars` to `pkg/core/info` (maybe character.go?) package with `SetupCharactersInCore` or change the type for Durability to `map[int][info.ReactionModKeyEnd]info.Durability`